src/node/delay.rs

```diff
@@ -24,15 +28,31 @@ impl Default for DelayOptions {
 }
 
 /// Node that delays the incoming audio signal by a certain amount
+/*
+ * For simplicity in the audio graph rendering, we have made the conscious decision to deviate from
+ * the spec and split the delay node up front in a reader and writer node (instead of during the
+ * render loop - see https://webaudio.github.io/web-audio-api/#rendering-loop )
+ *
+ * This has a drawback: a delay of 0 is no longer possible. This would only be possible if the
```

Comment:
"a delay of 0 is no longer possible, and the minimum delay is 128 samples (e.g. ~2.9ms at 44.1kHz)."

Reply:
Added to the public docstring 👍